pkg/apis/tensorflow/v1/doc.go

```diff
@@ -16,5 +16,5 @@
 // +k8s:openapi-gen=true
 
 // Package v1 is the v1 version of the API.
-// +groupName=kubeflow.org
+// +groupName=tensorflow.kubeflow.org
```

Comment:
It may be a block change /cc @kubeflow/wg-training-leads @zw0610

Reply:
Is it auto-generated or manually? I believe we should keep the group name, even in the documentation.

Reply:
> Is it auto-generated or manually? I believe we should keep the group name, even in the documentation.

This is generated manually, and the code-generator does not generate clientset.This is a workaround.More info in #1501